OSSM-8200 Document how to scope the mesh with Discovery Selectors (#84469)
openshift/openshift-docs: modules/ossm-scoping-service-mesh-with-discoveryselectors.adoc

// Module included in the following assemblies:
// install/ossm-installing-openshift-service-mesh.adoc

:_mod-docs-content-type: Concept
[id="ossm-scoping-service-mesh-with-discoveryselectors_{context}"]
= Scoping the Service Mesh with discovery selectors
:context: ossm-scoping-service-mesh-with-discoveryselectors

> A {SMProductShortName} includes workloads that meet the following criteria:

Reviewer: "A Service Mesh," "A service mesh," "Service Mesh," or other?

I see most frequently in published docs the common lowercase form when used with an article: https://docs.openshift.com/container-platform/4.17/service_mesh/v2x/ossm-create-mesh.html

* The control plane has discovered the workload.
* The workload has an Envoy proxy sidecar injected.

By default, the control plane discovers workloads in all namespaces across the cluster, so:

* Each proxy instance receives configuration for all namespaces, including workloads not enrolled in the mesh.
* Any workload with the appropriate pod or namespace injection label receives a proxy sidecar.

> In shared clusters, you may want to limit the scope of {SMProductShortName} to only certain namespaces. This approach is especially useful if multiple Service Meshes will run in the same cluster.

Reviewer 2: ```suggestion
In shared clusters, you might want to limit the scope of {SMProductShortName} to only certain namespaces. This approach is especially useful if multiple Service Meshes run in the same cluster.
```
- https://www.ibm.com/docs/en/ibm-style?topic=word-usage#may
- https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense

> In shared clusters, you might want to limit the scope of {SMProductShortName} to only certain namespaces. This approach is especially useful if multiple Service Meshes run in the same cluster.

Reviewer: Also verify "Service Meshes" vs. "service meshes."

Author: I will keep this in lower case in here as the product name is not being pointed out in here.
